### PR TITLE
Add dependabot support

### PR DESCRIPTION
**Pull request by notgull**
Adds support for dependabot in order to keep packages up to date.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: cargo
```
Comment (MarijnS95) [inline .github/dependabot.yml:3]: Don't forget to also add github-actions.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    directory: /
+    schedule:
+      interval: weekly
+    commit-message:
+      prefix: ''
+    labels: []
```
Comment (MarijnS95) [inline .github/dependabot.yml:9]: These seem to be the default and do not add value? Or is there special handling for `prefix: ''`? (I vaguely recall discussing this months ago)

Comment (notgull) [reply .github/dependabot.yml]: Looks like it; I've removed them accordingly.

**Comment by kchibisov** [comment]
I really don't like the dependabot, and given that  we have very limited amount of dependencies and we don't have any `lock` file we have nothing to update other than the `windows-sys`, and some macOS crates, both should be kept in sync with the winit anyway, so dependabot is sort of useless.

Given that we are a library the rest is bumped automatically.

**Comment by notgull** [comment]
> Given that we are a library the rest is bumped automatically.

What if I configured Dependabot to ignore minor/patch version bumps and to only make PRs for major version bumps?

**Comment by kchibisov** [comment]
Yeah, but it would be even better if it could list stuff in maintainers tab, and not even open more noise. Like they already have security alerts there from dependabot, could just add an extra to list the rest.

Though, issue is a way better, than stupid PR I'll close anyway.

**Comment by MarijnS95** [comment]
@kchibisov what do you think about revisiting this? It could remind us about for example the `libloading 0.9` bump, and various stale GitHub actions bumps. I have quite a nice terse setup to PR.

**Comment by kchibisov** [comment]
@MarijnS95 given that I don't have time to actively check anything for the time being, I think it's ok, given that glutin is low volume probably won't get lost.

**Pull request closed without merging**